### DISCUSSION TITLE
Clickable Player

### DISCUSSION THREAD

**Discussion by JPGuillemin** [z-huang/music#270] [Ideas]
Hi, 

In the "Player" view : the Artist Name and Album Name should be clickable to go directly to the Artist page or Album page :)

All the best
JP

**Comment by z-huang** [comment]
You can browse the artist or album in more action button.

**Reply by JPGuillemin** [reply]
Yes, but it's less, less, ergonomic

**Reply by z-huang** [reply]
I see that YT Music has this feature. I'll keep this in mind. I MAY implement it.

**Reply by JPGuillemin** [reply]
great :)

**Reply by z-huang** [reply]
Implemented.
